### DISCUSSION TITLE
Subdomain, platforms and mulitple projects

### DISCUSSION THREAD

**Discussion by linearshift** [vercel/community#1241] [Help] [answered]
### Summary

Hi!

I'm building subdomain-based profile pages using the platforms starter kit and all is going well, but I think I might have hit a problem.

In addition to the starter kit, I already have a separate Vercel project with a subdomain already configured and working - and I think this might cause a conflict.

Eg: - at the moment I have: 
widget.mydomain.com - pointing to an app within my monorepo

But now I also want to add another new project on mydomain.com, which uses middleware to rewrite the subdomains.

Looking at the examples I can see that one option is to move my widget project into a subfolder eg mydomain.com/pages/widget and handle the rewrite that way in the middleware file, but ideally I would like to keep the two projects separate. 

Is this possible? Can these two solutions for subdomains exist together?

Thanks in advance for any advice, tips or pointers in the right direction!

Tim

**Comment by amyegan** [comment, marked as answer]
The wildcard subdomain from your platforms starter kit project is hijacking all of your "mydomain.com" subdomains. You could do something with rewrites to disguise a separate domain, like in [How can I serve multiple projects under a single domain?](https://vercel.com/guides/how-can-i-serve-multiple-projects-under-a-single-domain#deploying-rewrite-rules)

**Comment by linearshift** [comment]
Ah - thanks, that's the clarification I was looking for! I will take a look at the link and see if that can solve the use case I have. Ta!
